src/backend/TrafficCourts/Citizen.Service/Services/Impl/CitizenDocumentService.cs

```diff
@@ -102,17 +102,21 @@ public async Task<List<FileMetadata>> FindFilesAsync(DocumentProperties properti
         {
             properties = new DocumentProperties(result.Metadata, result.Tags);
 
-            FileMetadata fileMetadata = new()
-            {
-                FileId = result.Id,
-                FileName = result.FileName,
-                DocumentType = properties.DocumentType,
-                NoticeOfDisputeGuid = properties.NoticeOfDisputeId?.ToString("d"),
-                VirusScanStatus = properties.VirusScanStatus,
-                DocumentStatus = properties.StaffReviewStatus,
-            };
-
-            fileData.Add(fileMetadata);
+            if (properties.DocumentSource is not DocumentSource.Staff) {                

```

Comment:
add this to line 51 and line 78

Reply:
Done. Threw an exception if the requested file is not a Citizen document.